code/game/objects/items/fireaxe.dm

```diff
@@ -70,11 +70,25 @@
 	base_icon_state = "bone_axe"
 	name = "bone axe"
 	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+//SKYRAT EDIT ADDITION BEGIN - Slight antimob axe buff
+	var/faction_bonus_force = 9 //Bonus force dealt against certain factions. Blatantly stolen from the arrow code, because shitcode
+	var/list/nemesis_factions = list("mining", "boss") //Any mob with a faction that exists in this list will take bonus damage/effects
 
 /obj/item/fireaxe/boneaxe/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded=5, force_wielded=23, icon_wielded="[base_icon_state]1")
 
+/obj/item/fireaxe/boneaxe/attack(mob/living/target, mob/living/carbon/human/user)
```

Comment:
Missing a `// SKYRAT ADDITION START` at the beginning.

Reply:
It's on the 73rd line?..

Reply:
Yeah it is and I've also added additional "/"'s (I forgot how you call those things) there.